src/CamOverlayDrawingAPI.ts

```diff
@@ -67,7 +66,6 @@ export class CamOverlayDrawingAPI extends EventEmitter {
     private callId: number;
     private sendMessages: Record<number, AsyncMessage>;
 
-    private connected = false;
     private ws?: WsClient;
```

Comment:
Možná bych zrušil možnost undefined a vytvářel WsClienta přímo v konstruktoru. V tuhle chvíli mě nenapadá jediný důvod, proč to mít zrovna takhle.

Reply:
opraveno